packages/workflows/src/handlers/middlewares/update-prodcut-reshape-sales-channes-data.ts

```diff
@@ -0,0 +1,44 @@
+import { WorkflowArguments } from "../../helper"
+import { UpdateProductsPreparedData } from "../product"
```

Comment:
suggestion: Are those middleware serving any purpose?

Reply:
yes, they are picking data from the prepared data step and reshaping them so the existing handler can be reused

Reply:
is it just renaming the alias?
I don't see any "reshape" happening here.

Reply:
Ahah i had the same thinking, but in fact it destructure the input data to grab a deeper object and assign it to the alias. It reshape in that sense but in fact it is more about reassigning to the alias a deep object instead of the root object

Reply:
I think a generic accessor middleware is good for such cases where you dont need to manipulate the data

Reply:
I agree, I initially went with that approach but had some issues with merge data and didn't want to block the PR with that. I think it would be a good thing to add, preferably in a separate PR with a TODO here for now, wdyt?

Reply:
Sounds good